### PR TITLE
Brief implementation skeleton for bool field type.

### PR DESCRIPTION
**Pull request by baldersheim**
@bratseth PR
This is just the basics needed for deployment to work.

### CODE REVIEW

**Comment by baldersheim** [comment]
I will merge to speed up end-2-end testing.

**Review by bratseth — COMMENTED** [review]
LGTM!

File: document/src/vespa/document/fieldvalue/boolfieldvalue.cpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (geirst) [inline document/src/vespa/document/fieldvalue/boolfieldvalue.cpp]: Please update year

File: document/src/vespa/document/fieldvalue/boolfieldvalue.h [hunk 2]
```diff
@@ -0,0 +1,37 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (geirst) [inline document/src/vespa/document/fieldvalue/boolfieldvalue.h]: Please update year

File: document/src/vespa/document/fieldvalue/boolfieldvalue.h [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+#pragma once
+
+#include "fieldvalue.h"
+
+namespace document {
+
+class BoolFieldValue : public FieldValue {
```
Comment (geirst) [inline document/src/vespa/document/fieldvalue/boolfieldvalue.h]: Please add class comment

**Review by geirst — COMMENTED** [review]
@baldersheim please have a look at my nitpick comments

**Pull request merged**